honorary-one-line-wonders/javascript/get_frequency.js

```diff
@@ -0,0 +1 @@
+const getFrequency=x=>{let p;x.sort();for(i=0,a=[],b=[];i<x.length;i++)x[i]!==p?(a.push(x[i]),b.push(1)):b[b.length-1]++,(p=x[i]);return a.map((v,i)=>({v,f:b[i]}))}
```

Comment:
This solution creates a function that is actually several lines long but just squashed "minified" with semicolons to replace a new line
This is a very cool concept, do you think it's possible to make this one line with [new Array()] and chained maps/reduce?